redux-form/lib/Field.d.ts

```diff
@@ -289,6 +295,17 @@ interface WrappedFieldMetaProps<S> {
     pristine: boolean;
 
     /**
+     * true if the field is currently being submitted
+     */
+    submitting?: boolean;
+
+    /**
+     * true if the form had onSubmit called and failed to submit for any reason.
+     * A subsequent successful submit will set it back to false.
+     */
+    submitFailed?: boolean;
+
```

Comment:
Prop `submitSucceeded` seems to be not present in the docs http://redux-form.com/6.5.0/docs/api/Field.md/